API/src/microservices/Assessment-Service/index.js

```diff
@@ -3,11 +3,31 @@ const { Assessments } = require(`../Database`);
 exports.submit = (assessment) => {
   return new Promise(async (resolve, reject) => { //eslint-disable-line
     try {
-      //use the bookshelf model Assessments from API/src/microservices/Database to save the assessment data in the PostgreSQL database
+      var date = new Date();
+      var shortDate = date.getFullYear()+'-' + (date.getMonth()+1) + '-'+date.getDate() 
+                      + ' '  +date.getHours()+ ':' + date.getMinutes()+ ':' + date.getSeconds();
```

Comment:
```suggestion
      const shortDate = `${date.getFullYear()} - ${date.getMonth()+1} - ${date.getDate()} 
                      ${date.getHours()}: ${date.getMinutes()}: ${date.getSeconds()}`;
```
In modern javascript, I think it's best practice to use const/let over the general "var".
I also recommend doing strings in script notation like this.  It's much cleaner because you don't have to use all the plus signs and manually add spaces.

Reply:
Thanks, I am going over all the suggestions and correcting my code.

Reply:
I changed this as recommended.